openmetadata-service/src/main/java/org/openmetadata/service/security/policyevaluator/PolicyCache.java

```diff
@@ -55,6 +50,13 @@ public static void initialize() {
     }
   }
 
+  public static PolicyCache getInstance() {
```

Comment:
Is the initialization done only once during system startup? If so, let's document it. Otherwise, this will have concurrency issues.